doc/README.librexgen

```diff
@@ -2,6 +2,12 @@ Rexgen.txt - This document describes how to use the rexgen library, to perform
 regex expression work within JtR.  Rexgen library is copywrite Jan Starke,
 jan.starke@outofbed.org   The regex.c code in JtR by JimF, Spring 2014.
```

Comment:
It's not currently called `Rexgen.txt`, and maybe `copyright`? Putting this together, with some other edits:

```
This document describes how to use the rexgen library to have JtR generate
candidate passwords matching a regular expression.   The rexgen library is by
Jan Starke.  The regex.c code in JtR is by JimF, Spring 2014.
```

Reply:
```
$ git blame regex.c|cut -b 13-26|sort|uniq -c|sort -nr
    252 JimF
    135 magnum  
     36 Frank Dittrich
     24 Jan Starke
$ git blame regex.c|grep Starke |head -n 1|cut -b 1-94
e40f1d3570 (JimF           2014-03-27 22:21:48 +0100   9)  * Author if this file is Jan Starke
```
IIRC, Jan Starke in 2014 provided the original Rexgen.txt which JimF renamed to README.librexgen, and he also provided the initial John the Ripper patch which JimF committed.
JimF among other things added support for --restore, magnum hybrid mode support, encoding changes, etc., Jan Starke later contributed adjustments to new libgexgen releases, I just contributed some minor fixes.

Reply:
OK, it's tricky. You want to leave this paragraph as-is for now?

Reply:
Yes, I don't want to be guilty of making matters worse.